gallery/how_to/work_with_microtvm/install_cmsis.rst

```diff
@@ -0,0 +1,30 @@
+..  Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+..    http://www.apache.org/licenses/LICENSE-2.0
+
+..  Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+Install CMSIS-NN
```

Comment:
should we add some clarifying text that this is for CoLab only, in case folks run across it as a standalone doc? or is it not linked from the tutorial index and won't be found through auto-indexing?

Reply:
This is not linked in the tutorial index and won't be found unless someone goes digging through the code. I'll add a comment though to clarify this.